RIS/astm-uam-psu-ris-0.0.0.yaml

```diff
@@ -0,0 +1,864 @@
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "429":
+          description: The client issued too many requests in a short period of time.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+      security:
+      - Authority:
+        - utm.strategic_coordination
+
+components:
+  schemas:
+    ResourceShape:
+      description: The physical shape of the resource.  This is not the same as the resource extent saved in the DSS. 
+      anyOf:
+        - $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Circle'
+        - $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/LatLngPoint'
+        - $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Polygon'
+    ResourcePhysicalDefinition:
+      required:
+        - name
+        - shape
+        - type
+      properties:
+        name:
+            description: Name of the resource
+            example: CHIPR
+            $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/TextShortType'
+        shape:
+          $ref: '#/components/schemas/ResourceShape'
+        altitude_lower:
+          description: Lower altitude limit of the resource, if any. 
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Altitude'
+        altitude_upper:
+          description: Upper altitude limit of the resource, if any. 
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Altitude'
+        type:
+          $ref: '#/components/schemas/ResourceType' 
+    ResourceStatus:
+      required: 
+        - status
+      properties:
+        status: 
+          type: string
+          description: The current status of the resource. 
+          enum: 
+          - ACTIVE
+          - INACTIVE
+    ResourceType:
+      required: 
+        - resource_type
+      properties:
+        resource_type: 
+          type: string
+          description: The type of resource.  
+          enum: 
+          - WAYPOINT
+          - VOLUME
+    CapacityLimitType:
+      required:
+        - capacity_type
+      type: object
+      properties:
+        capacity_type:
+          type: string
+          description: |
+            The type of capacity limitation which is to be met for the resource. 
+          externalDocs:
+            url: TBD
+          enum: 
+          - FLOW_RATE_CONSTANT
+          - FLOW_RATE_FIXED_BIN
+          - INSTANTANEOUS
+    CapacityPropertyType:
+      required:
+        - capacity_property
+        - capacity_value
+      type: object
+      properties:
+        capacity_property:
+          type: string
+          description: |
+            Property of the capacity check which must be met.
+            - RATE describes the number of crossings (or events) allowed per time interval DURATION (seconds).
```

Comment:
For INSTANTANEOUS, it's clear to me that only LIMIT can be specified. Please correct me if I'm wrong.

However, for FLOW_RATE_FIXED_BIN, DURATION is required. Do you expect both LIMIT and RATE to be specified in this case? If so, what would be the difference between them?